releases/ChangeLog.md

```diff
@@ -1,3 +1,8 @@
+## 5.1.3 - 2023-XX-XX
+
+- Bugfix: correct `scrptPubkey` parsing for segwit v1-v16
+- Bugfix: do not infer segwit just by availability of `PSBT_IN_WITNESS_UTXO` in PSBT
```

Comment:
forgot to add this line in previous [PR](https://github.com/Coldcard/firmware/pull/209)